pyproject.toml

```diff
@@ -7,6 +7,7 @@
 # SPDX-FileContributor: David Pape
 
 [tool.poetry]
+# Reference at https://python-poetry.org/docs/pyproject/
```

Comment:
```suggestion
```

I personally wouldn't add comments to pyproject.toml as this sometimes messes with things when the file is manipulated by Poetry. But I'm ok with leaving this line if you want.

Reply:
Oh I wouldn't have expected that a tool is messing with comments in the file... Not an attatool, comments are so useful!

I'm fine with whatever. WDYT @led02 @sdruskat ?